dotcom-rendering/src/web/components/StickyVideo.importable.tsx

```diff
@@ -0,0 +1,56 @@
+import { css } from '@emotion/react';
+import { useEffect, useState } from 'react';
+import { useStickyVideo } from '../lib/useStickyVideo';
+
+interface Props {
+	children: React.ReactNode;
+}
+
+const stuckStyles = css`
+	@keyframes fade-in-up {
+		0% {
+			opacity: 0;
+		}
+		100% {
+			transform: translateY(0);
+			opacity: 1;
+		}
+	}
+
+	height: 500px;
+	position: fixed;
+	bottom: 50px;
+	right: 20px;
+	width: 260px;
+	height: 145px;
+	z-index: 9999999;
```

Comment:
There’s a z-index helper method you can use instead of this!

[`getZIndex.tsx`](https://github.com/guardian/dotcom-rendering/blob/88d4102b6ad9a952822e71bf10be55bbed5f62ca/dotcom-rendering/src/web/lib/getZIndex.tsx)

Reply:
Didn't know about this, thanks!